diesel/src/query_dsl/group_by_dsl.rs

```diff
@@ -18,7 +18,7 @@ pub trait GroupByDsl<Expr: Expression> {
     type Output;
 
     /// See the trait documentation.
-    fn group_by(self, expr: Expr) -> dsl::GroupBy<Self, Expr>;
+    fn group_by(self, expr: Expr) -> Self::Output;
```

Comment:
Can we change that back? The main reason for the old version is that [rustdoc renders that as link to the correct type alias](https://docs.diesel.rs/2.0.x/diesel/query_dsl/methods/trait.GroupByDsl.html#tymethod.group_by), which is what any user should use as return type there. `Self::Output` might point to an non-nameable or will cause otherwise confusion.

Reply:
Yep, already changed back